### PR TITLE
Pre rails update changes

### PR DESCRIPTION
**Pull request by sonalkr132**
Removes some changes from 364285c0f52b49d60d24fb514f4c9fea2b386d0b on #1615

### CODE REVIEW

File: test/integration/password_reset_test.rb [hunk 1]
```diff
@@ -1,19 +1,25 @@
 require 'test_helper'
+include ActiveJob::TestHelper
 
 class PasswordResetTest < SystemTest
   setup do
     @user = create(:user, handle: nil)
+    @old_queue_adapter = queue_adapter
+    ActiveJob::Base.queue_adapter = :test
```
Comment (sonalkr132) [inline test/integration/password_reset_test.rb]: @simi 
> I think we can use test adapter as default one in test_helper.

I doubt we should tho. We don't always care if a job gets executed in test and using async adapter as default will make test faster.

Comment (simi) [inline test/integration/password_reset_test.rb]: Isn't job being executed only if it is enqueued in `perform_enqueued_jobs` block?

Comment (sonalkr132) [inline test/integration/password_reset_test.rb]: Yes. I would still trust rails to pick smart defaults. Can you please explain why do you think we should change it to `:test` everywhere?

Comment (sonalkr132) [inline test/integration/password_reset_test.rb]: From https://github.com/rails/rails/commit/625baa69d14881ac49ba2e5c7d9cac4b222d7022
>  Change the default adapter from inline to async. It's a better default as tests will then not mistakenly come to rely on behavior happening synchronously. This is especially important with things like jobs kicked off in Active Record lifecycle callbacks.
*DHH*

Comment (simi) [inline test/integration/password_reset_test.rb]: Maybe I'm wrong, but by some quick debugging I found this:

`ActiveJob::Base.queue_adapter` defaults to `:inline` in test now. That means every job is triggered inline and taking some time. So all triggered jobs in tests are triggered inline.

Here you're setting adapter to `:test` which doesn't trigger job inline by default, but you can change that by `perform_enqueued_jobs`.

So actually `:inline` and `:test` with `perform_enqueued_jobs` will result into same result here.

By using `:test` adapter by default, you can trigger jobs you only need by `perform_enqueued_jobs` or use another helpers even without job triggering (assert it's in queue only).

Comment (simi) [inline test/integration/password_reset_test.rb]: Just to notice, you're right @sonalkr132, this was changed in Rails 5.0.

**Comment by sonalkr132** [comment]
Changed queue_adapter to `:test`. It seem to affect only three tests in password test and doesn't affect over all test run time.

**Comment by simi** [comment]
Great!

**Pull request merged**